bin/mail/2016-03-21-teacher-recruitment/generate-ops-workshop-attendees

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require 'cdo/geocoder'
+
+# Teachers who attended Ops workshops, minus NYC
+
+teacher_query = <<eos
+  SELECT DISTINCT users.email, users.name, users.current_sign_in_ip FROM users
+    JOIN workshop_attendance ON (users.id = workshop_attendance.teacher_id)
+    JOIN segments ON (segments.id = workshop_attendance.segment_id)
+    JOIN workshops ON (workshops.id = segments.workshop_id)
+    WHERE workshops.location NOT LIKE '%NY%'
```

Comment:
Yes it would be in general, but in this case there are only 174 workshops at the moment and the only ones with NY in the location text are either "New York, NY" or "NY, NY" as seen from this query:

``` 
select distinct location from workshops where location like '%NY%'
```

None of them say NYC. I could change the query to NOT LIKE '%NY, NY%' AND NOT LIKE '%New York, NY%' but it gets the same results.